### ISSUE TITLE
Shorthand operator rule violation in unused files which are not added in XCode

### ISSUE THREAD

**Issue by userException** [realm/SwiftLint#3303]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

SwiftLint is reporting shorthand_operator violation rules in many of the Swift classes which are not added to my XCode project but are there in the project folder. Do I need to specify some temporary unused the files in "excluded" section for this?

<img width="1416" alt="Screenshot 2020-08-14 at 4 19 02 PM" src="https://user-images.githubusercontent.com/3499123/90242025-e12ca600-de49-11ea-95fd-93c442c23b4b.png">

### Pods
```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'



### YML
```
disabled_rules:
- trailing_whitespace
- vertical_whitespace
- trailing_newline
- colon
- comma
- control_statement
- identifier_name
- implicit_getter
opt_in_rules:
excluded:
- Carthage
- Pods
- SwiftLint/Common/3rdPartyLib
- CustomerPortal/Resources/JTAppleCalendar
force_cast: warning # implicitly
force_try:
  severity: warning # explicitly

line_length:
    warning: 400
    error: 450
    ignores_function_declarations: true
    ignores_comments: true
    ignores_urls: true
function_body_length:
    warning: 300
    error: 500
function_parameter_count:
    warning: 12
    error: 13
type_body_length:
    warning: 300
    error: 500
file_length:
    warning: 1000
    error: 1500
    ignore_comment_only_lines: true
type_name:
  min_length: 4 # only warning
  max_length: # warning and error
    warning: 40
    error: 50
  excluded: iPhone # excluded via string
  allowed_symbols: ["_"] # these are allowed in type names
identifier_name:
  min_length: # only min_length
    error: 2 # only error
  excluded: # excluded via string array
    - id
    - URL
    - GlobalAPIKey
reporter: "xcode"

```

### Environment

- Xcode: 11.6

**Comment by marcelofabri** [comment]
SwiftLint doesn't know anything about Xcode projects. You should use `included`/`excluded` in the configuration file or disable the rules locally in the file.
